content/events/project-pitches.yaml

```diff
@@ -0,0 +1,36 @@
+---
+  slug: project-pitches
+  name: Project Pitches
+  date: "2021-05-04T19:30:00+02:00"
+  description: |
+    We will received Project Pitches from you, and are super excited to hear you tell us about it. 
+    In this event, you'll present your idea to all techies, in a short, 2-3 minutes presentation 
+    (there are a lot of decks, so please be mindful of your presentation time!).
+
+    You will all have the change to make questions and later select your favorite 3 projects. We'll then assign the teams and get it all started!
+    
+    ## Project Pitch Decks
```

Comment:
You can remove this title, as we still do not have any project pitches ;)

Reply:
Done.